Shokofin/Providers/Helper.cs

```diff
@@ -15,100 +18,180 @@ public static string GetImageUrl(Image image)
             return image != null ? $"http://{Plugin.Instance.Configuration.Host}:{Plugin.Instance.Configuration.Port}/api/v3/Image/{image.Source}/{image.Type}/{image.ID}" : null;
         }
 
+        public static (int, int) GetNumbers(Models.Series series, Models.Episode.AniDB episode)
```

Comment:
Where is this function being used?

Reply:
Planned to use it in proposal I have. But forgot to remove it.